.NET/Microsoft.Recognizers.Text.DateTime/Turkish/Extractors/TurkishDatePeriodExtractorConfiguration.cs

```diff
@@ -328,9 +336,27 @@ public bool GetFromTokenIndex(string text, out int index)
         public bool GetBetweenTokenIndex(string text, out int index)
         {
             index = -1;
-            if (text.EndsWith("between"))
+            var match = RangePrefixRegex.MatchEnd(text, false);
+
+            if (match.Success)
+            {
+                index = match.Index;
+                return true;
+            }
+
+            string textTrm = text;
+            if (text.StartsWith("'"))
```

Comment:
What is this representing now?

Reply:
refactored this part to use only ExcludeSuffixRegex, its function is to exclude the suffix (if any) when looking for RangePrefixRegex matches